### ISSUE TITLE
How to change seed node

### ISSUE THREAD

**Issue by jackbond** [microsoft/service-fabric#1036]
I have a cluster with five seed nodes, and for whatever reason, SF demoted the 2nd one, and turned the 27th node into a seed node. How do I change it back? I'd like to scale the cluster down. Thanks.

**Comment by masnider** [comment]
Just scale it down. SF will handle the seed node reconfiguration automatically, or prevent the scale down if it would be unsafe to do so or something prevents it. This presumes you're on at least Silver durability.

**Comment by masnider** [comment]
Did you actually observe some issue when you tried this?

**Comment by jackbond** [comment]
Unfortunately I had to nuke the entire cluster because

Remove-ServiceFabricApplication 'ourAppname' -Force -ForceRemove

simply wouldn't remove the application (after several hours.) Our application creates and deletes lots of services on the fly, and if our cleanup routine fails, the number of services gets unwieldly, triggering low memory situations. Unfortunately, whatever SF code gets triggered by Remove-ServiceFabricApplication -ForceRemove isn't terribly forceful, as I have had to nuke entire clusters and restart from our RM script several times. Now I only see this in our dev clusters when I've intentionally caused this scenario, but your -ForceRemove code ought to instantly kill the processes and NOT allow them to be restarted, and I can guarantee you, that's NOT how it works right now.
